lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -136,6 +138,9 @@ export class Vulnerabilities {
             (event) => event.fixed !== undefined
           ).fixed,
           isVulnerabilityAlert: true,
+          force: {
```

Comment:
why this?

Reply:
Parity: https://github.com/renovatebot/renovate/blob/8bd77f747b3da59cc955b2fc4ba3301bcb680ea2/lib/workers/repository/init/vulnerability.ts#L222-L224